ckanext/ontario_theme/templates/internal/package/resource_validation.html

```diff
@@ -23,19 +30,79 @@
     </div>
     {% snippet "package/snippets/successful_page_alert.html", success_message=success_message %}
   {% else %}
+    {% set is_csv = "False" %}
     {% set success_message = "Data uploaded successfully" %}
     {% snippet "package/snippets/successful_page_alert.html", success_message=success_message %}
+    <a href="{{ action }}">
+      <button id="nextBtn"
+              class="ontario-button ontario-button--primary"
+              name="save"
+              type="submit">Continue</button>
+    </a>
```

Comment:
Interactive elements shouldn't be nested within one another for accessibility reasons. You can style the link as a button, add `role=button` and add javascript so the keydown and click events operate as expected or wrap the button in a form with the action being the link.